validators/validator.js

```diff
@@ -85,5 +85,12 @@ module.exports = (Generic) => {
                     });
             });
         },
+
+        //Todo: validation for description to avoid inserting sql statements etc.
+        // validateDescription(description) {
```

Comment:
Is this a vulnerability that would be added? Can you add sql statements in as is? I thought that the db.query method sanitized the input as it is a prepared statement but I could be wrong. If so, we need to do this kind of validation everywhere

Reply:
Yeah, I'm not sure, and after taking secure software last semester I'm thinking it may be a good idea to at least look into it.